pkg/engine/engine.go

```diff
@@ -84,6 +88,16 @@ func NewEngine(cfg *EngineConfig) (*Engine, error) {
 	e.dirs.src = srcdir
 	e.dirs.work = workdir
 
+	// load the .env.toml file.
+	e.env = new(api.EnvConfig)
+	path := filepath.Join(srcdir, ".env.toml")
+	switch _, err := toml.DecodeFile(path, e.env); err.(type) {
+	case nil:
+		fmt.Println("loading env from", path)
+	case *os.PathError:
+		// TODO no .env.toml file located, maybe print a warning.
```

Comment:
Wanna do this TODO before the merge? It's super simple

Reply:
Sure.